pyproject.toml

```diff
@@ -1,16 +1,17 @@
 [tool.pyright]
 
 # add any files/directories with type declaration to include
-include = ["pettingzoo/utils/env.py"]
+include = [
+    "pettingzoo/utils/env.py"
+]
 exclude = [
-    "**/node_modules",
-    "**/__pycache__",
 ]
-strict = ["pettingzoo/utils/env.py"]
-
-reportMissingImports = true
-reportMissingTypeStubs = false
+strict = [
+]
+# reportMissingImports = true
```

Comment:
Are these comments needed?

Reply:
This is still a work in progress, I'm not sure yet if they'll be needed or not in the end, but in any case I'll fix this before this PR is ready for review.